Source/WebCore/platform/graphics/FontSelectionAlgorithm.h

```diff
@@ -403,11 +399,6 @@ inline TextStream& operator<<(TextStream& ts, const std::optional<FontSelectionV
     return ts;
 }
 
-inline bool operator==(const FontSelectionRequest& a, const FontSelectionRequest& b)
-{
-    return a.tied() == b.tied();
```

Comment:
Maybe we should delete the `tied` function too and just move the code into the `add(Hasher&)` function?